### DISCUSSION TITLE
Can Sigma open Folder windows in the same place each time

### DISCUSSION THREAD

**Discussion by pebbleridge** [aleksey-hoffman/sigma-file-manager#326] [Q&A] [answered]
Hi,
I might work my windows differently to other people but I often more than one Folder at a time and copy/move files between them.
Windows (11) always opens the folder on top of each other and I have to move one so I can work with them both.
Will Sigma allow me to "save" the location that each folder opens in so I have them in the same place each time?
Regardfs,
Paul

**Comment by aleksey-hoffman** [comment, marked as answer]
Hi! This features should already work in v1, but I guess it's broken. I will fix it in v2.

Also, v2 already has the "split view" feature, that allows you to work with 2 folders at the same time.

![image](https://github.com/aleksey-hoffman/sigma-file-manager/assets/61761672/72f3b15b-6ebf-4f7e-bf60-ee254a63928b)

**Reply by pebbleridge** [reply]
Thanks for a quick reply.
I've not tried Sigma yet, I was asking the question before I installed it.
I'll download Sigma and give it a try :)

**Reply by aleksey-hoffman** [reply]
Cheers mate, let me know how it goes for you :)
